### PR TITLE
dark_plus: Add the borders color from the original theme

### PR DESCRIPTION
**Pull request by korken89**
Hi, II added the border for the `rulers` based on the border color of the original theme.
Without this the `rulers` are not shown.

### CODE REVIEW

File: runtime/themes/dark_plus.toml [hunk 1]
```diff
@@ -79,6 +79,7 @@
 "ui.text.focus" = { fg = "white" }
 
 "ui.virtual" = { fg = "dark_green" }
```
Comment (the-mikedavis) [inline runtime/themes/dark_plus.toml:81]: Do you have opinions on the new `ui.virtual` scope here? Trying out this change, I think

```toml
"ui.virtual" = { fg = "borders" }
```

Might be a better highlight. Wdyt? (`:set whitespace.render all` to turn on the whitespace rendering that uses it)

Comment (the-mikedavis) [reply runtime/themes/dark_plus.toml:81]: Ah that looks great! Would you mind adding that change here?

Comment (korken89) [reply runtime/themes/dark_plus.toml]: Fixed!

**Review by the-mikedavis — APPROVED** [review]
looks great, thanks!

**Comment by kirawi** [comment]
Where did you pull the color from? I can't seem to find it when running `Developer: Generate Color Theme From Current Settings` in VSCode.

**Comment by korken89** [comment]
@kirawi I found it here: https://github.com/samdark/intellij-visual-studio-code-dark-plus/blob/master/resources/visual_studio_code_dark_plus.theme.json#L10

**Pull request merged**